nova/requirements.yaml.tpl

```diff
@@ -2,12 +2,14 @@ dependencies:
   - name: postgres
     repository: http://localhost:8879/charts
     version: 0.1.0
-  - name: rabbitmq-notifications
-    repository: http://localhost:8879/charts
-    version: 0.0.1
   - name: utils
     repository: http://localhost:8879/charts
     version: 0.1.1
   - name: pg_metrics
     repository: http://localhost:8879/charts
     version: 0.1.0
+{{ if .Values.audit.enabled }}
+  - name: rabbitmq-notifications
+    repository: http://localhost:8879/charts
+    version: 0.0.1
```

Comment:
Actually, it was more supposed to be
```
  - name: rabbitmq-notifications
    repository: http://localhost:8879/charts
    condition: audit.enabled
    version: 0.0.1
```

Reply:
ok, this is more sophisticated than it used to be.

Updated PR is ready now.

Reply:
actually it is `condition: openstack.nova.audit.enabled` :-/

PR is ready